tests/integration/questionnaire/test_questionnaire_calculated_summary.py

```diff
@@ -61,24 +61,8 @@ def test_calculated_summary_no_skip(self):
             "We calculate the total of currency values entered to be £180.00"
         )
 
-    def test_new_calculated_summary(self):
-        self.launchSurvey("test_new_calculated_summary")
-        self._complete_calculated_summary_path_with_skip()
-        self.assertInBody("Skipped Fourth")
-        self.assertInBody(
-            "We calculate the total of currency values entered to be £80.00"
-        )
-
-    def test_new_calculated_summary_no_skip(self):
-        self.launchSurvey("test_new_calculated_summary")
-        self._complete_calculated_summary_path_no_skip()
-        self.assertNotInBody("Skipped Fourth")
-        self.assertInBody(
-            "We calculate the total of currency values entered to be £180.00"
-        )
-
-    def test_new_calculated_summary_repeating_section(self):
-        self.launchSurvey("test_new_calculated_summary_repeating_section")
+    def test_calculated_summary_repeating_section(self):
```

Comment:
The changes to these tests will need to be reverted

Reply:
Reverted this test file after bringing back the schema